en-US/firefox/new/horizon.lang

```diff
@@ -123,9 +130,9 @@ Firefox for other platforms
 Download Firefox <span>for Windows</span>
 
 
-## TAG: firefox_new_other_platforms
-;Download Firefox <span>for OS X</span>
-Download Firefox <span>for OS X</span>
+## TAG: macos_branding
```

Comment:
Or are you suggesting that the macos_branding behind all other firefox_new_other_platforms tags?